src/components/Combobox/Combobox.tsx

```diff
@@ -216,16 +217,25 @@ function Combobox<Item>({
                   {label}
                 </InputLabel>
               </InputControl>
-              <Icon
+              <Flex
```

Comment:
is there a solution that doesn't involve having a button? 

Currently the whole input is clickable. I personally wouldn't want to nest a second clickable element inside another.

Reply:
Hm...I couldn't come up with a different solution. I wanted to isolate the toggle of the results only to that specific icon so the user can click as many times as he wants in the search area without toggling the results.

Reply:
In the video i see the button changing vertical alignment wheny the dropdown is opened & closed? Can we make sure that it stays in the middle of the combobox?

Reply:
The arrow is staying at the middle of the combobox but the actual combobox height changes when the results are expanded. This happens because the clear all button is hidden when the Combobox is collapsed. This PR didn't affect the arrow position, I checked the master branch and the arrow has exactly the same behavior.